### PR TITLE
New configuration parameter 'tabs'

### PR DESCRIPTION
**Pull request by juaristi**
Adds a new configuration parameter 'tabs'. `tabs = true` will use tabs only, and `tabs = false` will use spaces only.

It works by setting/unsetting `expandtab`.

### CODE REVIEW

**Comment by juaristi** [comment]
Please, merge this, or tell me what is wrong.

Setting whether using tabs or spaces is useful since some code styles (mostly C code) mandate spaces while others mandate tabs (eg. kernel coding style). I code a lot in C and I work in many projects with different code styles, so I find very useful to be able to set for each project whether I want tabs or spaces. The length is then set with the already supported `indent` parameter.

File: autoload/exconfig.vim [hunk 1]
```diff
@@ -88,6 +88,13 @@ function exconfig#apply()
     let &tabstop=indent_stop
     let &shiftwidth=indent_stop
 
+    let my_et = vimentry#get('tabs')
```
Comment (jwu) [inline autoload/exconfig.vim]: how about 'expand_tab' instead of 'tabs' ?

**Comment by jwu** [comment]
Just comment the code. Would you mind to change the 'tabs' to 'expand_tab' ?

BTW: you also need to add the default value in ex-vimentry/autoload/vimentry.vim. looking forward to your PR 👍

**Comment by juaristi** [comment]
Hi,

I made the changes suggested. Parameter `tabs` is now called `expand_tab` and its value matches exactly `&expandtab` (formerly was just the opposite: tabs = true --> &expandtab = false).

I'll now send another PR to ex-vimentry setting its default value.

Tell me if you need something else. And sorry if my tone in the last comment seemed a bit rude, I didn't intend to. Vim is one of the few editors that don't suck, and exVim enhances it greatly.

**Comment by juaristi** [comment]
Done! The related ex-vimentry PR is exvim/ex-vimentry#3.

**Pull request merged**